.travis.yml

```diff
@@ -66,7 +76,6 @@ script:
 
 deploy:
   provider: script
-  skip_cleanup: true
```

Comment:
I believe this should remain as we need to build the `tar.gz` to ship to Ansible Galaxy and will fail on release if this option is removed.

Reply:
Travis build config validation is telling me "deploy: deprecated key skip_cleanup (not supported in dpl v2, use cleanup)". 

However I'm confused by this message now - it looks like it only applies to the travis dplv2 which isn't the default yet? Maybe I'll leave this in in-case it's still actually required in dplv1?
https://docs.travis-ci.com/user/deployment-v2#cleaning-up-the-git-working-directory

Reply:
I've added this line back in for now, as I haven't tested if the Travis deprecation warning is correct or not.